src/App.js

```diff
@@ -1,16 +1,49 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+
+  const initialTotalLikes = () => {
+    let likes = 0;
+    for (const message of messages) {
+      if (message.liked) {
+        likes += 1;
+      }
+    }
+    return likes;
+  };
+
+  const [totalLikes, setTotalLikes] = useState(initialTotalLikes);
```

Comment:
Small suggestion: in my working experience, developers set all their state variables at the top of the function component block, then their helper functions, then the return function.

I would consider moving this line to L20, right under the `const [messages, setMessages] = useState(chatMessages);`